### DISCUSSION TITLE
photoprism demo - photos are shown without addititional information?

### DISCUSSION THREAD

**Discussion by aisbergde** [photoprism/photoprism#3966] [Q&A]
I took a look on https://demo-de.photoprism.app/, and it is strange for me, that I can't get any information about the photo when I browse photos. For me, it seems to be very common, that it should be possible to get additional information (name, location, some other common metadata). Is this only missing in the demo, or is it in General a missing feature?

**Comment by lastzero** [comment]
You are probably missing a sidebar. However, you can see all that information and much more when you click the edit icon at the top. This is also described in our documentation and of course this is getting improved as our resources allow.

**Reply by aisbergde** [reply]
Exactly, I am missing a sidebar (Now I know how to call it, thank you). For years, I am using Google photos. And for me, it is common to quickly browse photos and getting some more information (especially geo information on a map) in a sidebar. Is this on the roadmap?

**Reply by lastzero** [reply]
We'll be developing a new viewer that provides it. Please see our existing GitHub issues.
